### ISSUE TITLE
"Upgrade connections to HTTPS" is on but shown as off

### ISSUE THREAD

**Issue by fabi-hub** [brave/brave-browser#15319]
## Description 
When I clear browsing data "Site and Shield Settings". The setting "Upgrade connections to HTTPS" is on again (like it should be), but in the Settings it is shown as disabled, while in Shield is shown as "on".

## Steps to Reproduce
 Clear browsing data and make sure "Site and Shield Settings" is checked.

When done the Settings looks like this:
![settings](https://user-images.githubusercontent.com/67245084/114912705-9d4ad000-9e20-11eb-86e4-01b0f378c4b6.png)

And in Shield it looks like this:
![shield](https://user-images.githubusercontent.com/67245084/114912738-a76cce80-9e20-11eb-874e-de7a9ee3da68.png)

## Expected result:
Orange dot in the settings

## Brave version (brave://version info)

Brave | 1.25.16 Chromium: 90.0.4430.70 (Official Build) nightly (64-bit)
-- | --
Revision | 3954de7175366f3b7edca576f140dfa273e6b5ae-refs/branch-heads/4430@{#1210}
OS | Windows 10 OS Version 2009 (Build 19042.906)
JavaScript | V8 9.0.257.16
User Agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.70 Safari/537.36

## Version/Channel Information:

I testet Nightly and Final, both have this bug

**Comment by rebron** [comment]
cc: @srirambv Can you double check this for me? I think this is a wontfix
@fabi-hub I can see the confusion. This is expected behavior since clearing browsing data for "Shields and Site Settings" clears the specific shield settings say for example a duckduckgo.com or other set of websites where you made changes. 

It's not going to reset settings on this page, that would be here: brave://settings/reset

**Comment by fabi-hub** [comment]
ah okay, I understand.

I thought, cleaning shield and site settings means they go back to the settings that I have selected in the main settings.

**Comment by srirambv** [comment]
@rebron  I think this is a bug. When we clear site settings it should reset all shields values to default (global and per site settings). So when you disable HTTPS and delete site settings via clear history its resetting to default but in brave://settings/shields its retaining the disabled state. Opening a new site shows HTTPS as enabled (which is correct). 

cc: @bridiver

**Comment by rebron** [comment]
@srirambv I agree this is a bug. Looks like there is an issue with the Block Scripts preference too. A clear of sites and shields settings should reset that setting too but it doesn't.

**Comment by bridiver** [comment]
@rebron the default for "Block Scripts" is disabled so that's correct

**Comment by bridiver** [comment]
I verified with http://https-everywhere.badssl.com/ that the default setting is working correctly and I have a theory of why this might be happening when you clear content settings. Discussing with @bbondy

**Comment by jeffmiko** [comment]
I do not have the issue when using an incognito window. I do have the issue with the normal mode.
